invokeai/frontend/web/src/features/system/components/AboutModal/AboutModal.tsx

```diff
@@ -0,0 +1,133 @@
+import { ExternalLinkIcon } from '@chakra-ui/icons';
+import {
+  Flex,
+  Grid,
+  GridItem,
+  Image,
+  Link,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { InvHeading } from 'common/components/InvHeading/wrapper';
+import {
+  InvModal,
+  InvModalBody,
+  InvModalCloseButton,
+  InvModalContent,
+  InvModalFooter,
+  InvModalHeader,
+  InvModalOverlay,
+} from 'common/components/InvModal/wrapper';
+import { InvText } from 'common/components/InvText/wrapper';
+import ScrollableContent from 'common/components/OverlayScrollbars/ScrollableContent';
+import {
+  discordLink,
+  githubLink,
+  websiteLink,
+} from 'features/system/store/constants';
+import { map } from 'lodash-es';
+import InvokeLogoYellow from 'public/assets/images/invoke-tag-lrg.svg';
+import type { ReactElement } from 'react';
+import { cloneElement, memo } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  useGetAppDepsQuery,
+  useGetAppVersionQuery,
+} from 'services/api/endpoints/appInfo';
+
+type AboutModalProps = {
+  /* The button to open the Settings Modal */
+  children: ReactElement;
+};
+
+const AboutModal = ({ children }: AboutModalProps) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { t } = useTranslation();
+  const { deps } = useGetAppDepsQuery(undefined, {
+    selectFromResult: ({ data }) => ({
+      deps: data ? map(data, (version, name) => ({ name, version })) : [],
+    }),
+  });
+  const { data: appVersion } = useGetAppVersionQuery();
+
+  return (
+    <>
+      {cloneElement(children, {
+        onClick: onOpen,
+      })}
+      <InvModal isOpen={isOpen} onClose={onClose} isCentered size="2xl">
+        <InvModalOverlay />
+        <InvModalContent maxH="80vh" h="80vh">
```

Comment:
The 80vh height is a bit too much - it means the modal can be way too tall.

Let's set the height to a value that shows all of the dependencies without scrolling, leaving the maxH where it's at.

Reply:
My screen size is 13 inch so its looking fine on my screen
<img width="1084" alt="Screenshot 2024-01-12 at 6 07 01 PM" src="https://github.com/invoke-ai/InvokeAI/assets/92542124/888e4bbe-f6dd-4165-9bca-e17af11bbd50">

Reply:
any height you suggest to go with?